tests/test_api.py

```diff
@@ -67,7 +69,7 @@ class TestCourseAccessGroupsViewSet(ViewSetTestBase):
 
     def test_sanity_check(self, client):
         assert self.user.is_active
-        assert list(self.my_org.sites.all()) == [self.site], 'There should be one organization site.'
+        assert get_organization_by_site(self.site) is not None, 'There should be linked site to the organization.'
```

Comment:
nitpick: `assert x` is the same as `assert x is not None` for this context.

```suggestion
        assert get_organization_by_site(self.site), 'There should be linked site to the organization.'
```